src/CoreHealthMonitor/CoreHealthMonitor.csproj

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <RuntimeIdentifier>win7-x64</RuntimeIdentifier>
+    <SelfContained>true</SelfContained>
```

Comment:
Why was this added? Isn't this running inside of Service Fabric?

Reply:
`For projects targeting .NET 8.0 and higher, specifying a RuntimeIdentifier will no longer produce a self contained app by default. To continue building self-contained apps, set the SelfContained property to true or use the --self-contained argument.`

After upgrade to 8.0 this warning is thrown for all projects that are using a `RuntimeIdentifier`. Enabling `--self-contained` for the entire build seemed to be wrong. Based on info in the docs and our repo just using the `SelfContained` prop I added it in the projects that were emitting the warning

Reply:
Ah, okay my bad. Had no idea `RuntimeIdentifier` meant self-contained in .NET 6.0

Reply:
Same 😅

Reply:
But not sure we're not changing the semantics for this one. If it was self-contained and runs inside of Helix.. No idea what will happen.

Reply:
Maybe you could check what's the artifacts published from the pipeline look like and pack the nugets locally too to see if there's a difference? E.g. .NET being bundled in one and not the other?

Reply:
Good point, will do

Reply:
Compared to latest available, the nuget structure is identical with no extra bits

![image](https://github.com/user-attachments/assets/c59a39ab-7e20-4fa2-b5b9-b4bf74773b42)

Reply:
Seems there is no difference compared to the latest available version

![image](https://github.com/user-attachments/assets/72647e6c-7b19-4619-8ea9-99594cb4d266)